### PR TITLE
Release/1.10.0

### PR DESCRIPTION
**Pull request by wow-such-code**

### CODE REVIEW

**Comment by wow-such-code** [comment]
[Prepare release of core-utils-lib 1.10.0](https://trello.com/c/Zn2xyuDL/976-prepare-release-of-core-utils-lib-1100)

File: CHANGELOG.rst [hunk 1]
```diff
@@ -5,6 +5,19 @@ Changelog
 This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
 
+1.10.0 (2021-09-10)
```
Comment (Steffengreiner) [inline CHANGELOG.rst:8]:
```suggestion
1.10.0 (2021-09-13)
```

**Review by Steffengreiner — CHANGES_REQUESTED** [review]
Heya @wow-such-code, thanks for preparing the release! 👍 
I found some minor issues that should be addressed before it's ready for merging. 

1. Instead of bumping the version by hand, i'd suggest using the qube bump-version functionality to keep the version in the qube files(such as qube.yml and qube.cfg) also in sync. 
2. This is a Nit but i'd appreciate it if the changelog date matches the release date

File: CHANGELOG.rst [hunk 2]
```diff
@@ -46,6 +59,18 @@ This project adheres to `Semantic Versioning <https://semver.org/>`_.
...
+
+**Added**
+
+**Fixed**
+
+**Dependencies**
+
+**Deprecated**
+
+
```
Comment (Steffengreiner) [inline CHANGELOG.rst:73]: I guess qube duplicated the Changelog entry? 😕 
```suggestion
```

**Review by Steffengreiner — CHANGES_REQUESTED** [review]
Nice Qube Bump-Version, however it duplicated an entry in the changelog. 🙈

**Review by Steffengreiner — APPROVED** [review]
LGTM :godmode:

**Pull request merged**